pkgs/development/libraries/protobuf/protobuf-cmake.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/cmake/install.cmake b/cmake/install.cmake
+index 9b2ae93cf..cef1f5c00 100644
+--- a/cmake/install.cmake
++++ b/cmake/install.cmake
+@@ -137,7 +137,7 @@ install(EXPORT protobuf-targets
+   NAMESPACE protobuf::
+   COMPONENT protobuf-export)
+ 
+-install(DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/${CMAKE_INSTALL_CMAKEDIR}/
++install(DIRECTORY ${CMAKE_INSTALL_CMAKEDIR}/
+   DESTINATION "${CMAKE_INSTALL_CMAKEDIR}"
```

Comment:
I suspect you are trying to deal with an absolute path we set in our default setup-hook:
https://github.com/NixOS/nixpkgs/blob/2ef110ae64a96c0d16df86f8390cfe0af7ff45dc/pkgs/development/tools/build-managers/cmake/setup-hook.sh#L81
But how can this work that you install from a directory in "$out" to itself.
I think, we typically workaround this by doing:
https://github.com/NixOS/nixpkgs/blob/2ef110ae64a96c0d16df86f8390cfe0af7ff45dc/pkgs/development/libraries/libfido2/default.nix#L45

Reply:
(see query above)